### DISCUSSION TITLE
24 of 74. so many false positive for latest version 22621.3880.66.4 given the pre-release version only has 7 of 74

### DISCUSSION THREAD

**Discussion by iwan1979** [valinet/ExplorerPatcher#3549] [General]
Hi,

First of all, thanks for all the great work on this app all this time.
I am excited knowing there is a latest version 22621.3880.66.4 been released on last week.
However, upon sending to virustotal for scanning, it detected as 24 of 74 false positives?
That's considered a lot compare to the pre-release version which only has 7 of 74 detection.
While the 22621.3880.66.4 for the amd64 only has 5 of 74 and the last stable version of 22621.3527.65.5 only has 1 of 75.

will the future stable release gonna has more and more false positive given the final version has 3x false positives compare to its pre-release or amd64?

Thanks again and have a great weekend ahead.

22621.3880.66.4:
https://www.virustotal.com/gui/file/a1a736c78531e03325a2beb70a4d802ba81adaae63f530c00fb8f8c96ce375ae?nocache=1
![20240809_230933](https://github.com/user-attachments/assets/7459ea3e-bbb5-46d4-a88c-ab24dbcf20c4)

22621.3880.66.4 pre-release:
https://www.virustotal.com/gui/file/2431cd45d5e5553a5b1e08e33750ed58bb34ee337aeae62790c9bae74abbc96a?nocache=1
![20240809_231920](https://github.com/user-attachments/assets/ddda99ad-82a4-4e65-a587-e8a462a3d538)

22621.3880.66.4 amd64:
https://www.virustotal.com/gui/file/48959062106efbb0b2f6b2e778640838feb015cb87420d8cf53dd92e7a9fcc37?nocache=1
![20240809_232058](https://github.com/user-attachments/assets/44693c7b-3d80-40a1-a24a-c19604721da3)

22621.3527.65.5:
https://www.virustotal.com/gui/file/6942de4845ca2bff1665cc1a0fb1a46eaa36404b7e4436dee9552ea5f2216f92?nocache=1
![20240809_232606](https://github.com/user-attachments/assets/738c048f-7c46-4744-a209-58fabefc7ad0)

**Comment by pyrates999** [comment]
Duplicate of: https://github.com/valinet/ExplorerPatcher/issues/3228

Please close this.

MS just doesn't like EP and has added it to windows defender, even though it is not a virus and not a hack tool.

You can also set windows defender to exclude the following:
1. C:\Program Files\ExplorerPatcher
2. %APPDATA%\ExplorerPatcher

Future updates to EP won't be flagged then.

You can also set windows defender to exclude the directory that you manually download EP to so you can install it without windows defender blocking it.

**Reply by iwan1979** [reply]
thanks @pyrates999 

Given this is open source project, my concern is with supply chain threat where any bad actors can insert malicious codes just like the case of solarwind n moveit. Hence whenever virustotal flagged over 10 detection, i'll be more cautious with whitelisting the app. 

I hope they can bring down the false positive given similar case #3122 was submitted in apr 2024 (https://github.com/valinet/ExplorerPatcher/discussions/3122#discussioncomment-9078862) for 22621.3296.64.3 and false positives of the subsequent releases were reduced.

Anyway, will close this for now

**Reply by pyrates999** [reply]
All builds are compiled using github servers.  You can also check all the commits yourself as it is open source.
